src/shared/components/search-result/search-result.component.ts

```diff
@@ -1,12 +1,14 @@
-import { Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges } from '@angular/core';
+import { ChangeDetectorRef, Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges } from '@angular/core';
 import { FormGroup } from '@angular/forms';
 import { AbstractAppConfig } from '../../../app.config';
 import { PlaceholderService } from '../../directives';
-import { CaseField, CaseState, CaseType, CaseView, DisplayMode,
+import {
+  CaseField, CaseState, CaseType, DisplayMode,
```

Comment:
This looks like an "interesting" auto-formatter. When a single line is too long, they should end up in a pure vertical list.